.env

```diff
@@ -0,0 +1,11 @@
+DOCKER_INFLUXDB_INIT_MODE=setup
+DOCKER_INFLUXDB_INIT_USERNAME= nuodb
+DOCKER_INFLUXDB_INIT_PASSWORD=helloworld
+DOCKER_INFLUXDB_INIT_ORG=nuodb
+DOCKER_INFLUXDB_INIT_RETENTION=1w
+DOCKER_INFLUXDB_INIT_BUCKET=telegraf
+DOCKER_INFLUXDB_INIT_ADMIN_TOKEN=quickbrownfoxjumpsoveralazydog
+NUOVERSION=5.0.1-2
```

Comment:
Can you bump this to a more recent version? I think 5.0.4 is the most recent version, but you should confirm.

Reply:
Would that affect the product version that is used on the internal test server or it will be overwritten regardless of the version set here?

Reply:
It will use the version set in the .env file